src/sentry/static/sentry/app/views/organizationStream/overview.jsx

```diff
@@ -370,16 +379,16 @@ const OrganizationStream = createReactClass({
       // if query is the same, just re-fetch data
       this.fetchData();
     } else {
-      this.transitionTo({query});
+      this.transitionTo(null, {query});
     }
   },
 
   onSortChange(sort) {
-    this.transitionTo({sort});
+    this.transitionTo(null, {sort});
```

Comment:
Should changing sort directions remove a saved search?

Reply:
Fixed + added a test